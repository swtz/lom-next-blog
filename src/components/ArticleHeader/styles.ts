import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Wrapper = styled.header`
  ${({ theme }) => css`
    padding-bottom: ${theme.spacings.xlarge};
    margin-bottom: ${theme.spacings.xlarge};
    border-bottom: 0.1rem solid ${theme.colors.mediumGray};

    ${HeadingStyles} {
      font-size: ${theme.font.sizes.xxlarge};
      margin: 0;
      margin-bottom: ${theme.spacings.medium};

      @media ${theme.media.lteMedium} {
        font-size: ${theme.font.sizes.large};
      }
    }
  `}
`;

export const Excerpt = styled.p`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.medium};
    margin: ${theme.spacings.medium} 0;

    @media ${theme.media.lteMedium} {
      font-size: 2rem;
    }
  `}
`;

export const Cover = styled.img`
  ${({ theme }) => css`
    max-width: 100%;
    display: block;
    margin-bottom: ${theme.spacings.medium};
  `}
`;
