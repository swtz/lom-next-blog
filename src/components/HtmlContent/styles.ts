import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    font-size: calc(${theme.font.sizes.xsmall} + 0.2rem);
    line-height: 1.5;

    p {
      margin: ${theme.spacings.medium} 0;
    }

    a,
    a:visited,
    a:link {
      color: ${theme.colors.secondary};
      text-decoration: none;
      transition: all 300ms ease-in-out;
    }

    a:hover {
      filter: brightness(50%);
    }

    code {
      font-family: monospace;
      color: ${theme.colors.secondary};
      font-size: ${theme.font.sizes.xsmall};
      background: ${theme.colors.mediumGray};
      padding: 0.2rem;
      margin: 0.2rem;
    }

    pre {
      background: ${theme.colors.primary};
      padding: ${theme.spacings.medium};
      font-family: monospace;
      color: ${theme.colors.white};
      margin: ${theme.spacings.medium} 0;
      font-size: ${theme.font.sizes.xsmall};
      width: 100%;
      overflow-x: auto;
    }

    pre code {
      color: inherit;
      background: inherit;
    }

    img {
      max-width: 100%;
    }

    .image {
      background: ${theme.colors.mediumGray};
      line-height: 0;
      margin: ${theme.spacings.medium} 0;
    }

    .image figcaption {
      background: ${theme.colors.mediumGray};
      font-size: ${theme.font.sizes.xsmall};
      padding: ${theme.spacings.small};
      text-align: center;
      line-height: 1.3;
    }

    .image-style-side {
      float: right;
      max-width: 50%;
      margin: ${theme.spacings.medium} ${theme.spacings.small};
    }

    hr {
      border: none;
      border-bottom: 0.1rem solid ${theme.colors.mediumGray};
    }

    ul,
    ol {
      margin: ${theme.spacings.medium} ${theme.spacings.xlarge};
    }

    .table {
      width: 100%;
      overflow: hidden;
      overflow-x: auto;
    }

    table {
      width: 100%;
      border-collapse: collapse;
      margin: ${theme.spacings.medium} 0;
    }

    table td,
    table th {
      padding: ${theme.spacings.small};
      border: 0.1rem solid ${theme.colors.mediumGray};
    }

    .media iframe {
      width: 100%;
      height: auto;
      aspect-ratio: 16 / 9;
    }

    blockquote {
      border-left: 0.5rem solid ${theme.colors.secondary};
      color: ${theme.colors.darkerGray};
      filter: brightness(80%);
      padding-left: ${theme.spacings.medium};
      font-style: italic;
      margin: ${theme.spacings.medium};
    }

    @media ${theme.media.lteMedium} {
      font-size: 2rem;

      pre {
        font-size: 1.2rem;
      }

      .image-style-side {
        float: none;
        max-width: 100%;
        margin: ${theme.spacings.xlarge} 0;
      }
    }
  `};
`;
