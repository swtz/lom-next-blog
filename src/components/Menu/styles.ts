import styled, { css } from 'styled-components';

export const Wrapper = styled.div`
  ${() => css``}
`;

export const Nav = styled.nav`
  ${() => css``}
`;

export const Logo = styled.div`
  ${() => css``}
`;

export const OpenClose = styled.a`
  ${() => css``}
`;
