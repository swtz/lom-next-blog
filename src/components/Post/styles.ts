import styled, { css } from 'styled-components';

export const Wrapper = styled.article`
  ${() => css``}
`;

export const Container = styled.div`
  ${() => css``}
`;
