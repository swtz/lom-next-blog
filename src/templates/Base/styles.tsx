import styled, { css } from 'styled-components';

export const Wrapper = styled.div`
  ${() => css``}
`;
export const HeaderContainer = styled.div`
  ${() => css``}
`;
export const ContentContainer = styled.div`
  ${() => css``}
`;
export const FooterContainer = styled.div`
  ${() => css``}
`;
