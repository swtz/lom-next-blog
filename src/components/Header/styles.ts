import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Wrapper = styled.header`
  ${({ theme }) => css`
    display: flex;
    justify-content: center;
    align-items: center;
    /* padding: ${theme.spacings.xhuge}; */
    width: 100%;
    max-width: ${theme.sizes.max};
    color: ${theme.colors.darkText};
    font-size: ${theme.font.sizes.xsmall};
    margin: 0 auto;

    ${HeadingStyles} {
      margin: 0 0 calc(${theme.spacings.xsmall} - 1rem);
    }

    @media ${theme.media.lteXSmall} {
      flex-flow: column wrap;

      > div {
        ${HeadingStyles} {
          margin: ${theme.spacings.xsmall} 0
            calc(${theme.spacings.small} - 1rem);
        }
      }
    }
  `}
`;

export const Content = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-flow: column wrap;
    margin-left: ${theme.spacings.large};
    max-width: 48rem;
  `}
`;
