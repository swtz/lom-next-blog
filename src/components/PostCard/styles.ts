import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Wrapper = styled.div`
  text-align: center;

  ${({ theme }) => css`
    ${HeadingStyles} {
      margin: 0;
      margin-top: ${theme.spacings.small};
      margin-bottom: ${theme.spacings.small};
      transition: all 300ms ease-in-out;
    }

    a {
      text-decoration: none;
    }

    &:hover ${HeadingStyles} {
      color: ${theme.colors.secondary};
    }

    &:hover img {
      opacity: 0.8;
    }
  `}
`;

export const Cover = styled.img`
  ${() => css`
    max-width: 100%;
    transition: opacity 300ms ease-in-out;
  `}
`;
export const Excerpt = styled.p`
  ${() => css``}
`;
