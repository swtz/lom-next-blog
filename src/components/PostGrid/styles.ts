import styled, { css } from 'styled-components';
import { Wrapper as PostCardStyles } from '../PostCard/styles';

export const Wrapper = styled.div`
  ${() => css``}
`;

export const Grid = styled.div`
  ${({ theme }) => css`
    display: grid;
    justify-content: center;
    align-items: center;
    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
    gap: 3rem;
    padding: 1rem;

    ${PostCardStyles} {
      padding: 1rem;
      border: 0.5rem solid ${theme.colors.secondary};
      border-radius: 5%;
      height: 100%;
      transition: border-color 300ms ease-in-out;
    }

    ${PostCardStyles}:hover {
      border-color: ${theme.colors.darkText};
    }
  `}
`;
