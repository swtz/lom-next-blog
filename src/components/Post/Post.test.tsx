import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import { Post, PostProps } from '.';

import mockHeader from '../Header/mock';
import mockArticleHeader from '../ArticleHeader/mock';
import mockHtmlContent from '../HtmlContent/mock';

const props: PostProps = {
  ...mockHeader,
  ...mockArticleHeader,
  content: mockHtmlContent,
};

describe('<Post />', () => {
  it('should render elements from the Header, ArticleHeader, and HtmlContent components inside Post.', () => {
    const spreadProps = { ...props };
    renderTheme(<Post {...spreadProps} />);

    expect(
      screen.getByRole('heading', { name: 'Blog do Arroz Ã© vida!' }),
    ).toBeInTheDocument();

    expect(screen.getByAltText(props.excerpt)).toBeInTheDocument();

    expect(screen.getByRole('blockquote')).toBeInTheDocument();
  });

  it('should render elements from the Header, ArticleHeader, and HtmlContent components inside Post.', () => {
    const spreadProps = { ...props };
    const { container } = renderTheme(<Post {...spreadProps} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
