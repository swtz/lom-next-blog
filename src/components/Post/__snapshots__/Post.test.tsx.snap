// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Post /> should render elements from the Header, ArticleHeader, and HtmlContent components inside Post. 1`] = `
.c13 {
  font-size: 2.4rem;
}

.c13 p {
  margin: 4.0rem 0;
}

.c13 a,
.c13 a:visited,
.c13 a:link {
  color: #DC143C;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c13 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c13 pre {
  background: #000000;
  padding: 2.4rem;
  font-family: monospace;
  color: #FFFFFF;
  margin: 4.0rem 0;
  font-size: 1.6rem;
  width: 100%;
  overflow-x: auto;
}

.c13 img {
  max-width: 100%;
}

.c13 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 4.0rem 0;
}

.c13 .image figcaption {
  background: #DDDDDD;
  font-size: 1.6rem;
  padding: .8rem;
  text-align: center;
  line-height: 1.3;
}

.c13 .image-style-side {
  float: right;
  max-width: 50%;
  margin: 2.4rem;
}

.c13 hr {
  border: none;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c13 ul,
.c13 ol {
  margin: 4.0rem;
}

.c13 .table {
  width: 100%;
  overflow: hidden;
  overflow-x: auto;
}

.c13 table {
  width: 100%;
  border-collapse: collapse;
}

.c13 table td,
.c13 table th {
  padding: .8rem;
  border: 0.1rem solid #DDDDDD;
}

.c13 .media iframe {
  width: 100%;
  height: auto;
  aspect-ratio: 16 / 9;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
  max-width: 96rem;
  margin: 0 auto;
}

.c0 .c12 {
  width: 100%;
  max-width: 80rem;
}

.c0 .c12 p:first-child {
  margin-top: 0;
}

.c11 {
  font-size: 1.6rem;
  color: #AAAAAA;
  font-style: italic;
}

.c11 .categories span::after {
  content: ', ';
}

.c11 .categories span:last-child:after {
  content: '';
}

.c11 a {
  color: #DC143C;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c11 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c3 {
  color: #000000;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c6 {
  color: #000000;
  font-size: 2.4rem;
  text-transform: none;
}

.c8 {
  color: #000000;
  font-size: 6.4rem;
  text-transform: none;
}

.c7 {
  padding-bottom: 4.0rem;
  margin-bottom: 4.0rem;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c7 .c2 {
  font-size: 4.8rem;
  margin: 0;
  margin-bottom: 2.4rem;
}

.c9 {
  font-size: 2.4rem;
  margin: 2.4rem 0;
}

.c10 {
  max-width: 100%;
  display: block;
  margin-bottom: 2.4rem;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c4 > img {
  width: 15rem;
  height: 15rem;
  border-radius: 50%;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 6.4rem;
  width: 100%;
  max-width: 96rem;
  color: #333333;
  font-size: 1.6rem;
  margin: 0 auto;
}

.c1 .c2 {
  margin: 0 0 calc(1.6rem - 1rem);
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column wrap;
  -ms-flex-flow: column wrap;
  flex-flow: column wrap;
  margin-left: 3.2rem;
  max-width: 48rem;
}

@media (max-width:768px) {
  .c13 {
    font-size: 2rem;
  }

  .c13 pre {
    font-size: 1.2rem;
  }

  .c13 .image-style-side {
    float: none;
    max-width: 100%;
    margin: 4.0rem 0;
  }
}

@media (max-width:768px) {
  .c8 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c7 .c2 {
    font-size: 3.2rem;
  }
}

@media (max-width:768px) {
  .c9 {
    font-size: 2rem;
  }
}

@media (max-width:550px) {
  .c1 {
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
  }

  .c1 > div .c2 {
    margin: 1.6rem 0 calc(.8rem - 1rem);
  }
}

<div
  class="c0"
>
  <header
    class="c1"
  >
    <h1
      class="c2 c3"
    >
      <a
        class="c4"
        href="/"
        target="_self"
      >
        <img
          alt="Blog do Arroz é vida! - Rice is a cereal grain and in its domesticated form is the staple food of over half of the world's population, particularly in Asia and Africa. Rice is the seed of the grass species Oryza sativa."
          src="/assets/images/logo.svg"
        />
      </a>
    </h1>
    <div
      class="c5"
    >
      <h2
        class="c2 c6"
      >
        Blog do Arroz é vida!
      </h2>
      <p>
        Rice is a cereal grain and in its domesticated form is the staple food of over half of the world's population, particularly in Asia and Africa. Rice is the seed of the grass species Oryza sativa.
      </p>
    </div>
  </header>
  <header
    class="c7"
  >
    <h2
      class="c2 c8"
    >
      Hello, rice! With rice is possible make a several delicious dishes
    </h2>
    <p
      class="c9"
    >
      Rice is a cereal grain and in its domesticated form is the staple food of over half of the world's population, particularly in Asia and Africa. Rice is the seed of the grass species Oryza sativa (Asian rice).
    </p>
    <img
      alt="Rice is a cereal grain and in its domesticated form is the staple food of over half of the world's population, particularly in Asia and Africa. Rice is the seed of the grass species Oryza sativa (Asian rice)."
      class="c10"
      src="/assets/images/arroz-branco-saudavel.webp"
    />
    <div
      class="c11"
    >
      <p>
        <span>
          Por 
        </span>
        <a
          href="/author/surfista"
        >
          surfista
        </a>
        <span
          class="separator"
        >
           | 
        </span>
        <time
          datetime="2025-02-12T11:41:13.991Z"
        >
          12 de fev. de 2025
        </time>
        <span
          class="separator"
        >
           | 
        </span>
        <span
          class="categories"
        >
          <span>
            <a
              href="/category/off-topic"
            >
              Off Topic
            </a>
          </span>
          <span>
            <a
              href="/category/tech"
            >
              tech
            </a>
          </span>
        </span>
      </p>
    </div>
  </header>
  <div
    class="c12 c13"
  >
    <p>
      Nostro 
      <strong>
        definitiones
      </strong>
       (
      <i>
        React
      </i>
      ) sed cu - 
      <u>
        Otávio Miranda
      </u>
      . Sea 
      <code>
        useEffect
      </code>
      , useState dicat fabulas 
      <a
        href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
      >
        consetetur
      </a>
       no, odio deseruisse assueverit mei eu, vis probo nonumes molestie id. Ipsum movet liberavisse an mea, diceret vituperata ne his, delenit apeirian quo ea. Imperdiet percipitur te eos. Per ludus aliquam cu. 
      <a
        href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
      >
        Eos ex tincidunt necessitatibus, timeam theophrastus mel cu
      </a>
      . La.
    </p>
    


    <h1>
      Vida de uma maneira que a sua presença não seja notada, mas que sua ausência seja sentida.
    </h1>
    


    <h2>
      Tincidunt necessitatibus, timeam theophrastus mel cu
    </h2>
    

    <h3>
      Tincidunt necessitatibus, timeam theophrastus mel cu
    </h3>
    

    <h4>
      Tincidunt necessitatibus, timeam theophrastus mel cu
    </h4>
    

    <h5>
      Tincidunt necessitatibus, timeam theophrastus mel cu
    </h5>
    

    <h6>
      Tincidunt necessitatibus, timeam theophrastus mel cu
    </h6>
    


    <pre>
      <code
        class="language-plaintext"
      >
        import { HtmlContent } from '../HtmlContent';
import * as Styled from './styles';

export type FooterProps = {
  footerHtml: string;
};

export const Footer = ({ footerHtml }: FooterProps) =&gt; {
  return (
    &lt;Styled.Container&gt;
      &lt;HtmlContent html={footerHtml} /&gt;
    &lt;/Styled.Container&gt;
  );
};

      </code>
    </pre>
    



    <p>
      Qui eu 
      <i>
        mutat
      </i>
       exerci 
      <span
        style="background-color:hsl(60,75%,60%);"
      >
        dolores
      </span>
      , his no 
      <span
        style="color:hsl(270,75%,60%);"
      >
        repudiare
      </span>
       constituam 
      <s>
        accommodare
      </s>
      , vis ea sint 
      <u>
        tempor
      </u>
       eirmod. In mei eruditi moderatius, ex per iudico essent repudiandae. In vis vidisse menandri, an has repudiare disputationi delicatissimi. In has wisi justo postulant. Ei viderer ornatus mel, usu an partiendo tincidunt, ea eum brute ullamcorper.
    </p>
    <p>
      10
      <sup>
        2
      </sup>
      *5
      <sup>
        2
      </sup>
       - H
      <sub>
        2
      </sub>
      0
    </p>
    <p
      style="text-align:center;"
    >
      <strong>
        Center: 
      </strong>
      Abhorreant reformidans eum an, per equidem repudiandae no, ex cum elitr argumentum persequeris. In eam consul theophrastus delicatissimi, lobortis rationibus efficiendi mea ei, nam natum dolore scripserit ea. Sit partiendo eloquentiam ne. Sea no autem dolores, ridens gloriatur mel cu. Munere nullam epicuri his id, dicam definiebas deterruisset eu vim. Ea veritus deleniti usu, nam ei idque choro volumus, ex nisl aliquam bonorum eam.
    </p>
    <p>
      Fugit aliquip erroribus eu eam, an mel vidit doctus. Vis nobis veritus maluisset id, nonumy adversarium neglegentur est ut. Tation persius pri eu, modo minim fastidii mei no. Ad tota pertinax definiebas eam.
    </p>
    <p
      style="text-align:right;"
    >
      <strong>
        Right:
      </strong>
       Mel in lorem feugiat, tation dolore et eum. Id has utamur albucius molestiae. Dico justo maiestatis ei mei, labitur laboramus voluptatum nam ut. Saperet propriae nec an.
    </p>
    <p>
      Quod oblique mei ne, cetero delicatissimi sea cu, cu est harum dicit verterem. Tota forensibus ut usu, luptatum dignissim persecuti vel ex. Mea nostrud instructior cu, et referrentur theophrastus eum. Pro in iuvaret senserit. In persius explicari conceptam nec, in populo ponderum consequat vis. Gloriatur conclusionemque no qui, vitae nominati nec te, laoreet torquatos accommodare ex qui. Eu natum mediocritatem eos, cum ex alia oportere.
    </p>
    <p
      style="text-align:justify;"
    >
      <strong>
        Justify:
      </strong>
       Cu vis nonumes petentium disputationi, invidunt aliquando ex ius, ocurreret suscipiantur conclusionemque eu qui. Probo nemore ut est. Et eam dicta oratio eligendi. Quem dicta dolore vel et, at reque gloriatur definitionem sed.
    </p>
    <p>
      Nec simul ullamcorper at. Mea possit dolores persecuti te, ei quidam apeirian quo. Has quas corpora ne. Ius ad malis iusto delenit, sit ferri idque ex.
    </p>
    <blockquote>
      <p>
        Blockquote: Et sed legere rationibus, vitae evertitur ad eam. Eu amet vero officiis vis. Ad nullam pericula neglegentur sit, vix ei regione legendos argumentum. Mei an quas convenire, nobis dicunt mei ut. Pro no debet referrentur, ei est falli libris. Eu per summo salutandi hendrerit, salutatus democritum interesset nec ut. Wisi lobortis incorrupte ei mea, explicari interpretaris has et.
      </p>
    </blockquote>
    <p>
      A new line:
    </p>
    <hr />
    <p>
      Now keep going…
    </p>
    <figure
      class="image"
    >
      <a
        href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
      >
        <img
          alt="O alt text da imagem"
          src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
        />
      </a>
      <figcaption>
        Aqui está uma imagem
      </figcaption>
    </figure>
    <p>
      Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
    </p>
    <figure
      class="image image-style-side"
    >
      <img
        alt="Uma imagem à direita"
        src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
      />
      <figcaption>
        Uma imagem à direita
      </figcaption>
    </figure>
    <p>
      Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
    </p>
    <p>
      Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
    </p>
    <p>
      Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
    </p>
    <p>
      Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
    </p>
    <p>
      Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
    </p>
    <p>
      Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
    </p>
    <p>
      Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
    </p>
    <p>
      Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
    </p>
    <p>
      Bullet:
    </p>
    <ul>
      <li>
        Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
      </li>
      <li>
        iriure posidonium consectetuer ad eam.
      </li>
      <li>
        Nobis reprimique duo in, eam corpora mandamus
      </li>
      <li>
        ex. In
      </li>
      <li>
        quod euismod
      </li>
      <li>
        eum, ne habemus cotidieque necessitatibus cum.
      </li>
      <li>
        Liber recteque duo no, ludus reprehendunt has an.
      </li>
    </ul>
    <p>
      Now ordered:
    </p>
    <ol>
      <li>
        Item 1
      </li>
      <li>
        Item
      </li>
      <li>
        Item
      </li>
      <li>
        Item
      </li>
    </ol>
    <p>
      A think that's it…
    </p>
    <figure
      class="table"
    >
      <table>
        <thead>
          <tr>
            <th>
              Title 1
            </th>
            <th>
              Title 1
            </th>
            <th>
              Title 1
            </th>
            <th>
              Title 1
            </th>
            <th>
              Title 1
            </th>
            <th>
              Title 1
            </th>
            <th>
              Title 1
            </th>
            <th>
              Title 1
            </th>
            <th>
              Title 1
            </th>
            <th>
              Title 1
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
          </tr>
          <tr>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
          </tr>
          <tr>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
            <td>
              Item
            </td>
          </tr>
        </tbody>
      </table>
    </figure>
    <p>
      A video:
    </p>
    <figure
      class="media"
    >
      <iframe
        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
        allowfullscreen=""
        frameborder="0"
        height="315"
        referrerpolicy="strict-origin-when-cross-origin"
        src="https://www.youtube.com/embed/LVShoJjFcdo?si=65W8zKDI3_VxVkfq"
        title="YouTube video player"
        width="560"
      />
    </figure>
    <p>
      The end.
    </p>
  </div>
</div>
`;
